default.nix

```diff
@@ -121,6 +121,18 @@ let
     };
   };
 
+  haskell-nix-extra-packages =
+    let haskellNix = import sources."haskell.nix";
+    in with (import sources.nixpkgs {
+    inherit system crossSystem;
+    config = haskellNix.config // config;
+    overlays = haskellNix.overlays ++ overlays.haskell-nix-extra;
+  }); {
+    inherit
+      regenerateStackPackages
```

Comment:
Why is this not `stackProjectRegenerate` to be in line with `cabalProjectRegenerate`?

Reply:
There was indeed an old `cabal-project-regenerate.nix` file but I just deleted it (not used). 
The new `cabalProjectRegenerate` is completely different than `regenerateStackPackages`: it only update `--sha256` entries in `cabal.project`.

Reply:
please rename to stackProjectRegenerate